core/src/main/java/org/elasticsearch/action/main/MainResponse.java

```diff
@@ -75,6 +81,7 @@ public void writeTo(StreamOutput out) throws IOException {
         out.writeString(nodeName);
         Version.writeVersion(version, out);
         clusterName.writeTo(out);
+        out.writeString(clusterUUID);
```

Comment:
I like the rules for naming variables [here](https://google.github.io/styleguide/javaguide.html#s5.3-camel-case) which'd have this named `clusterUuid`. We certainly haven't standardized on those rules, but I do like them.